src/components/banner/MasterBanner.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Image1 from "../../assets/images/mockup.png";
+import { IoIosCall } from "react-icons/io";
+import { IoMdMail } from "react-icons/io";
+const MasterBanner = () => {
+  return (
+    <>
+      {" "}
+      <div className="flex w-full gap-12  p-4 ">
```

Comment:
wrap this content with `<saection>` tag instead of `<div>` 
```suggestion
      <section className="flex w-full gap-12  p-4 ">
```